### PR TITLE
Extend the Description of the AssetLink Request

### PR DESCRIPTION
**Pull request by sebbader-sap**

### CODE REVIEW

File: DiscoveryServiceSpecification/V3.0_SSP-001.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ paths:
         - $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.1.0#/components/parameters/Limit'
         - $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.1.0#/components/parameters/Cursor'
       requestBody:
-        description: A list of specific Asset identifiers
+        description: A list of asset identifiers in the form of an AssetLink. One AssetLink may either represent tehn name/value pair of a SpecificAssetId or the globalAssetId of the asset represented by the Asset Administration Shell.
```
Comment (BirgitBoss) [inline DiscoveryServiceSpecification/V3.0_SSP-001.yaml:62]:
```suggestion
        description: A list of asset identifiers in the form of an AssetLink. One AssetLink may either represent the name/value pair of a SpecificAssetId or the globalAssetId of the asset represented by the Asset Administration Shell.
```

Comment (BirgitBoss) [reply DiscoveryServiceSpecification/V3.0_SSP-001.yaml:62]: Typo

**Review by BirgitBoss — CHANGES_REQUESTED** [review]
Correct typos

**Comment by sebbader-sap** [comment]
@BirgitBoss typo is corrected, please check again.

**Review by BirgitBoss — APPROVED** [review]

**Pull request merged**